### ISSUE TITLE
Incorrect Test Case - 987. Vertical Order Traversal of a Binary Tree

### ISSUE THREAD

**Issue by stellentus** [LeetCode-Feedback/LeetCode-Feedback#8925]
#### Your LeetCode username
`lightbulb`

#### Category of the bug
- [ ] Question
- [ ] Solution
- [ ] Language
- [X] Missing Test Cases (actually incorrect)

#### Description of the bug
A test case sorts in the wrong order.

The problem says "There may be multiple nodes in the same row and same column. In such a case, sort these nodes by their values."

It then gives example outputs showing the values are sorted in *ascending* order: `[[9],[3,15],[20],[7]]` and `[[4],[2],[1,5,6],[3],[7]]`.

However, I get the following from one of the test cases:
```
Input:    [3,1,4,0,2,2]
Output:   [[0],[1],[2,2,3],[4]]
Expected: [[0],[1],[3,2,2],[4]]
```

You can see the expected output is incorrectly sorted in *descending* order.

#### Code you used for Submit/Run operation
I can provide code if necessary, but this should be straightforward to verify even without code

#### Language used for code
golang

**Comment by fenstea** [comment]
Hi James, 
I thought I'll answer since I just worked through this case yesterday.
So in this case, 2, 2 and 3 are in the same group because they are in the same column, but they are on a different levels of the tree:
--> [(3, 0), (2, 2), (2, 2)]
3 is root so it's at the beginning of the column (level 0 of the tree) and then there are two elements at the same level and in the same column, so we sort them by values (and in this case they are identical).

I circled back to the description and I think this part highlights the idea:
_The vertical order traversal of a binary tree is **a list of top-to-bottom orderings** for each column index_

Let me know if that was helpful!
